flintrock/config.yaml.template

```diff
@@ -1,6 +1,7 @@
 services:
   spark:
     version: 1.6.0
+    # preferred-mirror: # optional; default to 'https://s3.amazonaws.com/spark-related-packages/${file}'
```

Comment:
As proposed in #71, I prefer `{v}` or even `{version}` instead of `${file}`, since the latter's exact meaning to the user is not clear.